ledger-tool/src/main.rs

```diff
@@ -1692,6 +1718,13 @@ fn main() {
             ("genesis", Some(arg_matches)) => {
                 println!("{}", open_genesis_config_by(&ledger_path, arg_matches));
             }
+            ("genesis-accounts", Some(arg_matches)) => {
```

Comment:
I was thinking it could be an option under "genesis" potentially: `solana-ledger-tool genesis --accounts`

I see how it might fit under "accounts" from a cli usage perspective, implementation wise this might get a little clunky though

Reply:
>implementation wise this might get a little clunky though

Yeah, good point. Intuitively, I think someone will think of "genesis" as the main "topic" first, so grouping it here makes sense.

This would be for another PR, and I'm not intentionally trying to change CLI up, but adding `accounts` as an arg under genesis would make me argue that `genesis-hash` should be under there too